### PR TITLE
Message to be displayed if the word doesn't exist

### PR DESCRIPTION
**Pull request by KeerthanBhat**
I have tried to return None from the method where 'Nonetype isn't iterable' error was being shown.

### CODE REVIEW

**Comment by kshitij10496** [comment]
Can you be a bit more clear about what you mean by "word doesn't exist"?
Does this mean that a word which doesn't exist in the API dictionary?

Also, can you share what would like to be displayed when the user enters such a word?

**Comment by KeerthanBhat** [comment]
Yeah, for the words which doesn't exist in the API dictionary. "Word doesn't exist. Please check the spelling." can be displayed.

File: lexico/cli.py [hunk 1]
```diff
@@ -24,9 +24,13 @@ def add(word):
     except ConfigFileError:
         click.echo('You need to initialize the application.')
         click.echo('Run:\n\t\t $ lexico init')
+    #else:   
+    #    click.echo_via_pager(word_data.stringify())
     else:
-        click.echo_via_pager(word_data.stringify())
-
+	if word_data is None:
```
Comment (kshitij10496) [inline lexico/cli.py]: There seems to be inconsistent use of tabs and spaces here.

**Comment by KeerthanBhat** [comment]
Changed the spaces. How about this?

File: lexico/cli.py [hunk 2]
```diff
@@ -24,10 +24,11 @@ def add(word):
     except ConfigFileError:
         click.echo('You need to initialize the application.')
         click.echo('Run:\n\t\t $ lexico init')
+    elif word_data is None:
```
Comment (kshitij10496) [inline lexico/cli.py:27]: I am facing `SyntaxError` here.

**Comment by kshitij10496** [comment]
You should always test your changes before committing.
Generally, most projects have a test suite which assures that nothing breaks.
However, we currently do not have one.

Here is how you can test the changes manually:
From the base folder of the project:
`$ pip3 install -e .`
This will install the package in development mode and track all the changes you make.
Now, you can manually try commands which test your fixes.
For example, in this case you can try to enter wrong words and see if the message is displayed.

**Comment by kshitij10496** [comment]
Updates @KeerthanBhat

**Comment by KeerthanBhat** [comment]
I'm trying to debug my error. Couldn't do much from 3-4 days since my exams have started. 
I will surely fix this patch and submit new patches post 21st.

**Comment by kshitij10496** [comment]
Thanks for the heads up.
Do remember to fill up the midterm evaluation by tomorrow.

**Comment by KeerthanBhat** [comment]
`pip3 install -e` is showing errors. Typed this from base directory itself.

**Comment by kshitij10496** [comment]
Try: `pip3 install -e .` ; Dont forget the dot '.'

Also, does pip3 refer to the Python3 package installer?
Try `$which pip3` and try to debug the error you are getting.

**Comment by KeerthanBhat** [comment]
Okay. Got it. Thanks.
What features do we get in development mode?
Why don't we get the error while not using the development mode?

**Comment by kshitij10496** [comment]
In development mode, all the changes you make to the source code are reflected in the command.
Try adding couple of `print` statements in the `lexico` function of the `cli.py` module.
You will be able to figure it out from then on.

**Pull request closed without merging**